Nx errors when run in a workspace generated by the Angular CLI

## Expected Behavior

Some kind of output which indicates that Nx is aware it can't be used in the current workspace. Possibly even a message which tells the user how to upgrade the current workspace to an Nx workspace?

At the very least, _not_ an error.

## Current Behavior

```bash
▶ nx
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '<cut for brevity>/node_modules/@nrwl/cli/bin/nx.js'
```

### Steps to Reproduce

Run `nx` in a workspace generated by the Angular CLI.

e.g. 

`ng generate new-workspace --create-application=false`
`cd new-workspace`
`nx`

This is far too easily done if you're playing with Angular CLI and Nx workspaces side by side :)

--- a/packages/cli/lib/init-global.ts
+++ b/packages/cli/lib/init-global.ts
@@ -1,4 +1,5 @@
 import chalk from 'chalk';
+import * as fs from 'fs';
 import * as path from 'path';
 import { findWorkspaceRoot } from './find-workspace-root';
 import { output } from './output';
@@ -11,14 +12,34 @@ export function initGlobal() {
 
   if (workspace) {
     // Found a workspace root - hand off to the local copy of Nx
-    require(path.join(
+    const localNx = path.join(
       workspace.dir,
       'node_modules',
       '@nrwl',
       'cli',
       'bin',
       'nx.js'
-    ));
+    );
+    if (fs.existsSync(localNx)) {
+      require(localNx);
+    } else {
+      if (fs.existsSync(path.join(workspace.dir, 'node_modules'))) {
+        output.error({
+          title: `Could not find Nx in this workspace.`,
+          bodyLines: [
+            `To convert an Angular workspace to Nx run: ${chalk.bold.white(
+              `ng add @nrwl/workspace`
+            )}`,
+          ],
+        });
+      } else {
+        output.error({
+          title: `Could not find a node_modules folder in this workspace.`,
+          bodyLines: [`Have you run ${chalk.bold.white(`npm/yarn install`)}?`],
+        });
+      }
+      process.exit(1);
+    }
   } else {
     output.log({
       title: `The current directory isn't part of an Nx workspace.`,

